package.json

```diff
@@ -9,6 +9,17 @@
         "start": "concurrently --kill-others \"cd lib/destack && npm run dev\" \"cd dev/nextjs-project && npm run dev\""
```

Comment:
Add the `prepare` script? (Got it from #14 )
```js
"prepare": "husky install"
```

Reply:
Ahh, forgot about that

Reply:
No worries!